src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -64,6 +64,13 @@ public static void ImaginaryOne()
             VerifyMagnitudePhaseProperties(Complex.ImaginaryOne, 1, Math.PI / 2);
         }
 
+        [Fact]
+        public static void SqrtMinusOne()
+        {
+            // Sqrt(-1) should be I exactly, not just tollerably close.
```

Comment:
And please disable this test for netfx (SkipOnTargetFrameworkAttribute). Add a netfx specific test for bonus points :)